sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/AzureTableAsyncClient.java

```diff
@@ -0,0 +1,32 @@
+package com.azure.data.tables;
+
+import reactor.core.publisher.Mono;
+
+import java.time.Duration;
+import java.util.HashSet;
+import java.util.Set;
+
+public class AzureTableAsyncClient {
+    private final Set<String> existingTables = new HashSet<>();
+
+    /**
+     * Creates a table. This artificially takes 3 seconds.
+     *
+     * @param name The name of the table.
+     *
+     * @return A Mono that completes when the table is created.
+     */
+    public Mono<AzureTable> createTable(String name) {
+        return Mono.delay(Duration.ofSeconds(3)).flatMap(delay -> {
+            if (existingTables.add(name)) {
+                System.out.printf("CREATING TABLE '%s'.%n", name);
```

Comment:
This should be using ClientLogger rather than System.err. or System.out. If you look at some of the azure-storage-blob client classes, you'll see examples of this.